docs/error-reporting/platform-integrations/source-map.md

```diff
@@ -9,88 +9,259 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-The following steps guide you through configuring your JS application to automatically upload sourcemap files during the project build. Additionally, it explains how to set the appropriate ID in your Backtrace client to match errors with the corresponding sourcemap files.
+The following steps guide you through configuring your JS application to automatically upload sourcemap files during the project build.
 
 ## What You'll Need
 
 - A Backtrace account ([log in](https://backtrace.io/login) or sign up for a [free trial license](https://backtrace.io/sign-up)).
 - Your subdomain name (used to connect to your Backtrace instance). For example, `https://example-subdomain.sp.backtrace.io`.
-- A Backtrace project and a submission token.
+- A Backtrace project, an error submission token and a symbol submission token.
 
-:::tip Generate a Submission Token
+:::tip Generating Tokens
+## Generating an Error Submission Token
 
 1. In the Backtrace Console, go to **Project settings > Error submission > Submission tokens**.
 1. Select **+**.
 
+
+## Generating a Symbol Submission Token
+
+1. In the Backtrace Console, go to **Project settings > Symbols > Access tokens**.
+1. Select **+**.
+
 :::
 
-## Step 1: Generate Source Maps for Your Application
+## Obtaining Symbol Submission URL
+
+If you're using a hosted instance of Backtrace, you most likely need only to pass the subdomain name. You can resolve your subdomain name from your instance address.
+
+For example, if your instance address is `https://example.sp.backtrace.io`, your subdomain will be `example`.
+
+If, for some reason, you cannot upload sourcemaps using this way, or you're using an on-premise installation, retrieve the whole URL using the following steps.
+
+### Hosted Instance
+
+If your instance is hosted on backtrace.io, you can create the URL using `https://submit.backtrace.io/<your_subdomain>/<submission_token>/sourcemap`.
+
+For example, for subdomain `https://example.sp.backtrace.io` and token `bebbbc8b2bdfac76ad803b03561b25a44039e892ffd3e0beeb71770d08e2c8a7`, the URL will be `https://submit.backtrace.io/example/bebbbc8b2bdfac76ad803b03561b25a44039e892ffd3e0beeb71770d08e2c8a7/sourcemap`.
+
+### On-premise
 
-This step is application-specific. Take note of the location of your build folder as you'll need it in step 4.
+If your instance is hosted on-premise, you can create the URL using `<your address>:6098//post?format=sourcemap&token=<submission token>`.
```

Comment:
This detail is not exclusively pertinent to source maps. The user will have already has to resolve their universe name in the initial integration, prior to generating source maps. Showing this detail here detracts, I think, from the main task.

Options:
* Move this detail, which is really useful, to a separate doc that we can reference from multiple locations
* Keep it here in this doc, but remove it from the main flow of generating source maps